tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -460,13 +466,17 @@ describe('DocumentationTopic', () => {
     const primary = wrapper.find(PrimaryContent);
     expect(primary.props('sections')).toEqual(propsData.primaryContentSections);
     const declarationContainer = wrapper.find('.declarations-container');
-    expect(declarationContainer.classes()).not.toContain('minimized-container');
+    // expect(declarationContainer.classes()).not.toContain('minimized-container');
     expect(declarationContainer.find(Declaration).props()).toEqual({
       conformance: propsData.conformance,
       declarations: declarationsSection.declarations,
       source: propsData.remoteSource,
     });
-    wrapper.setProps({ enableMinimized: true });
+    // wrapper.setProps({ enableMinimized: true });
+    // commented this out and moved it to the above `setProps` call because
+    // there seems to be an obscure bug with vue-test-utils where things don't
+    // work right if `setProps` is called more than once with a prop that is
+    // also used in the component's `provide`...
```

Comment:
🤔 `setProps` wont update the `provide`, as it's not reactive by design I guess?

Reply:
Yeah, Hanqing pointed that out too, and that makes sense. However, I was seeing weird results where some things were treated as if the flag was true and others as if it was false—it was very confusing 😅